app/src/sharedTest/java/org/oppia/android/app/profile/ProfileChooserFragmentTest.kt

```diff
@@ -712,6 +717,191 @@ class ProfileChooserFragmentTest {
     }
   }
 
+  @Test
+  @Config(qualifiers = "land")
+  fun testFragment_enableOnboardingV2_ltr_checkListIsAlphabetical() {
+    TestPlatformParameterModule.forceEnableOnboardingFlowV2(true)
+    profileTestHelper.addOnlyAdminProfile()
+    profileTestHelper.addMoreProfiles(9)
+    launch(ProfileChooserActivity::class.java).use {
+      testCoroutineDispatchers.runCurrent()
+
+      it.onActivity {
+        onView(
+          withViewAtPosition(
+            recyclerViewId = R.id.profiles_list_landscape,
+            position = 0,
+            targetViewId = R.id.profile_name_text,
+            itemMatcher = allOf(withText("Admin"), isDisplayed())
```

Comment:
This doesn't technically verify that it's alphabetical, just that admin is first. Do we already verify elsewhere the whole list is populated correctly?

Reply:
I have refactored the test to verify the position of all the created profiles.